frontend/static/quotes/code_java.json

```
@@ -46,8 +46,56 @@
     {
       "text": "public enum Wood {\n\tINDIAN_ROSEWOOD, BRAZILIAN_ROSEWOOD, MAHOGANY,\n\tMAPLE, COCOBOLO, CEDAR, ADIRONDACK, ALDER, SITKA;\n\n\tpublic String toString() {\n\t\tswitch(this) {\n\t\t\tcase INDIAN_ROSEWOOD: return \"Indian Rosewood\";\n\t\t\tcase BRAZILIAN_ROSEWOOD: return \"Brazilian Rosewood\";\n\t\t\tcase MAHOGANY: return \"Mahogany\";\n\t\t\tcase MAPLE: return \"Maple\";\n\t\t\tcase COCOBOLO: return \"Cocobolo\";\n\t\t\tcase CEDAR: return \"Cedar\";\n\t\t\tcase ADIRONDACK: return \"Adirondack\";\n\t\t\tcase ALDER: return \"Alder\";\n\t\t\tcase SITKA: return \"Sitka\";\n\t\t\tdefault: return \"Any\";\n\t\t}\n\t}\n}",
       "id": 7,
-      "source": "Head First Objiect-Oriented Analysis and Design",
+      "source": "Head First Object-Oriented Analysis and Design",
       "length": 545
+    },
+    {
+      "text": "public static int fibonacci(int n) {\n\tif (n <= 2) {\n\t\treturn 1;\n\t} else {\n\t\treturn fibonacci(n - 1) + fibonacci(n - 2);\n}",
+      "id": 8,
+      "source": "Community Contribution: Find the nth Fibonacci Number Recursively",
```

Should I remove the Community Contribution prefix to make it shorter? Likewise for the quotes below.

```suggestion
      "source": "Find the nth Fibonacci Number Recursively",
```

---

Remove those